frontend/public/components/storage/create-pvc.tsx

```diff
@@ -0,0 +1,353 @@
+                required
+              />
+            </div>
+            <p className="form-text text-muted" id="pvc-name-help">
+              A unique name for the storage claim within the project.
+            </p>
+            <div className="form-group">
+              <label className="control-label co-required" htmlFor="mount-path">
+                Access Mode
+              </label>
+              <div className="form-display-flex">
+                {this.state.radios.map(radio => {
+                  const checked = radio.value === this.state.accessMode;
+                  return (
+                    <RadioInput
+                      {...radio}
+                      key={radio.value}
+                      onChange={this.handleChange}
+                      checked={checked}
+                      aria-describedby="mount-path-help"
+                      className="horizontal-radio-group-text"
+                      id="accessMode"
+                      name="accessMode"
+                    />
+                  );
+                })}
+              </div>
+              <p className="form-text text-muted" id="access-mode-help">
+                Permissions to the mounted drive.
+              </p>
+            </div>
+            <label className="control-label co-required" htmlFor="volume-name">
+              Size
+            </label>
+            <RequestSizeInput
+              name="requestSize"
+              required={false}
+              onChange={this.handleChange}
+              placeholder={defaultRequestSizeValue}
+              defaultRequestSizeUnit={defaultRequestSizeUnit}
+              dropdownUnits={dropdownUnits}
+            />
+            <p className="form-text text-muted" id="requestSize-help">
+              Desired storage capacity.
+            </p>
+            <div>
+              <Checkbox
+                value="Use label selectors to request storage"
+                onChange={this.onShowLabelSelectors}
+                checked={showKeyValue}
+                name="showLabelSelector"
+              />
+            </div>
+            <div className="form-group">
+              {this.state.showKeyValue && (
+                <NameValueEditor
+                  nameValuePairs={nameValuePairs}
+                  valueString="Selector"
+                  nameString="Label"
+                  addString="Add Value"
+                  readOnly={false}
+                  allowSorting={false}
+                  updateParentData={this.handleNameValuePairs}
+                />
+              )}
+              <p className="form-text text-muted" id="requestSize-help">
+                Use label selectors to define how storage is created.
+              </p>
+            </div>
+          </div>
+          <ButtonBar errorMessage={error} inProgress={inProgress}>
+            <button type="submit" className="btn btn-primary" id="save-changes">
+              {this.props.saveButtonText || 'Create'}
```

Comment:
Since the form is only used for creating pvc then I dont think that OR statement is needed

Reply:
I envision this being used for edit in the near future but I can remove it for now.